cypress/fixtures/mockUser.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Maybe @mkrumholz @redferret or @JoePeterson51 can give us a better signature/shape for this data

Reply:
I just deleted this one since it's not in their wiki yet